pages/src/_data/buildingsettings.yml

```diff
@@ -40,4 +40,29 @@ glassblower:
       - name: Priority
         description: This is the default setting. The hut will try to use the recipe that is higher up in their recipe list first.
       - name: Warehouse Stock
-        description: The hut will look in the warehouse first to see what resource you have more of before deciding what recipe it will use.
\ No newline at end of file
+        description: The hut will look in the warehouse first to see what resource you have more of before deciding what recipe it will use.
+guardtower:
+  - name: Task
+    description: This is where you can choose if you want the Guard to patrol, follow, or guard.
+    options:
+      - name: Patrol
+        description: The guard will patrol an area you designate in <strong>Patrol Settings</strong>. 
+      - name: Guard
+        description: You can set one area for the Guard to stay in. When you click "Set Target", you will be taken to the <strong>Selection Tools</strong> tab, where you can get the the Guard Scepter and designate the guard location.
+      - name: Follow
+        description: The guard will follow you around as your personal bodyguard, protecting you or fighting alongside you. They will even go outside the colony when following! You can designate how they follow you in <strong>Follow Settings</strong>. 
+      - name: Patrol Mine
+        description: The Guard will patrol the level of a <a href="../../source/buildings/mine">Mine</a> the <a href="../../source/workers/miner">Miner</a> is currently mining at. You must also select the guard in the Mine GUI in order for the guard to being patrolling.
+  - name: Retreat on low health
+    description: Here you can choose if the Guard will retreat when they have low health, if possible. You can choose from Off or On.
+  - name: Hire trainee
+    description: If there is a vacancy at this tower, here is where you can choose if you want a Guard to be hired from the respective training facility (<a href="../../source/buildings/archery">Archery</a> for Archers and <a href="../../source/buildings/combatacademy"> Combat Academy</a> for knights) instead of an unemployed colonist. (This setting only matters if Assign Colonists to Jobs is turned to Automatic in the <a href="../../source/buildings/townhall"> Town Hall</a> GUI.)
```

Comment:
This is slightly inaccurate. This setting also applies if the hiring setting at this guard tower is Automatic, regardless of what the town hall says. Perhaps it can be shortened to something like:

>  description: If there is a vacancy at this tower, a new <a href="../../source/workers/guard">Knight or Archer</a> can be hired from the respective training facility (<a href="../../source/buildings/archery">Archery</a> for Archers and <a href="../../source/buildings/combatacademy"> Combat Academy</a> for Knights) instead of an unemployed colonist. This setting only applies if Automatic Hiring is turned on for this tower.